easypy/caching.py

```diff
@@ -141,12 +141,17 @@ def inner(*args, **kwargs):
             ret = func(*args, **kwargs)
             self.set(key, ret)
             return ret
+
+        inner = timecache(expiration=self.expiration)(inner)
+        self._timecache_clear = inner.cache_clear
```

Comment:
Maybe I'm not following this correctly, but wouldn't this override `self._timecache_clear` to be the `cache_clear` of the _last_ returned cache? If the `PersistentCache` is called multiple times, `clear()` will only clear the cache of the last call?